cli/src/semgrep/meta.py

```diff
@@ -593,15 +621,39 @@ def _fetch_branch_get_merge_base(branch_name: str, head_sha: str) -> str:
 
     @property
     def repo_name(self) -> str:
-        return os.getenv("CI_PROJECT_PATH", "[unknown]")
+        project_path = os.getenv("CI_PROJECT_PATH")
```

Comment:
Note that os.getenv second arg is a default value if things are not set. So this can probably be rewritten as:
```
return os.getenv("SEMGREP_REPO_NAME", os.getenv("CI_PROJECT_PATH"))
```